### ISSUE TITLE
Erlang script crashes when rebar version is 3.0.0+build.272.ref7d29b74

### ISSUE THREAD

**Issue by tank-bohr** [artemeff/jwt#1]
Вот так вот
```
===> Evaluating config script "_build/default/lib/jwt/rebar.config.script"
===> Uncaught error in rebar_core. Run with DEBUG=1 to see stacktrace
===> Uncaught error: {badmatch,
                             {error,
                              {14,file,
                               {error,
                                {badmatch,
                                 ["3","0","0+build","272","ref7d29b74"]},
                                [{erl_eval,expr,3,[]}]}}}}
===> Stack trace to the error location: [{rebar_config,consult_file_,
                                                 1,
                                                 [{file,
                                                   "/home/travis/build/rebar/rebar3/_build/default/lib/rebar/src/rebar_config.erl"},
                                                  {line,106}]},
                                                {rebar_config,consult_file,1,
                                                 [{file,
                                                   "/home/travis/build/rebar/rebar3/_build/default/lib/rebar/src/rebar_config.erl"},
                                                  {line,90}]},
                                                {rebar_app_utils,dep_to_app,
                                                 7,
                                                 [{file,
                                                   "/home/travis/build/rebar/rebar3/_build/default/lib/rebar/src/rebar_app_utils.erl"},
                                                  {line,164}]},
                                                {rebar_app_utils,
                                                 '-parse_deps/6-lc$^0/1-0-',6,
                                                 [{file,
                                                   "/home/travis/build/rebar/rebar3/_build/default/lib/rebar/src/rebar_app_utils.erl"},
                                                  {line,98}]},
                                                {rebar_app_utils,
                                                 '-parse_deps/6-lc$^0/1-0-',6,
                                                 [{file,
                                                   "/home/travis/build/rebar/rebar3/_build/default/lib/rebar/src/rebar_app_utils.erl"},
                                                  {line,98}]},
                                                {rebar_app_discover,
                                                 '-do/2-fun-0-',3,
                                                 [{file,
                                                   "/home/travis/build/rebar/rebar3/_build/default/lib/rebar/src/rebar_app_discover.erl"},
                                                  {line,31}]},
                                                {lists,foldl,3,
                                                 [{file,"lists.erl"},
                                                  {line,1262}]},
                                                {rebar_app_discover,do,2,
                                                 [{file,
                                                   "/home/travis/build/rebar/rebar3/_build/default/lib/rebar/src/rebar_app_discover.erl"},
                                                  {line,27}]}]
```

**Comment by artemeff** [comment]
solved with f4ce31caca8c3a52e828cad29fa7205eab47ca12

**Comment by tank-bohr** [comment]
Thanks :smile_cat:
